python/lsst/ts/cRIOpy/M1M3FCUTable.py

```diff
@@ -0,0 +1,130 @@
+# This file is part of M1M3 GUI.
+#
+# Developed for the LSST Telescope and Site Systems.  This product includes
+# software developed by the LSST Project (https://www.lsst.org).  See the
+# COPYRIGHT file at the top-level directory of this distribution for details of
+# code ownership.
+#
+# This program is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later
+# version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["FCUTABLE", "actuatorAddressToIndex"]
+
+FCUTABLE_INDEX = 0
+FCUTABLE_XPOSITION = 1
+FCUTABLE_YPOSITION = 2
+FCUTABLE_ZPOSITION = 3
+FCUTABLE_ADDRESS = 4
+
+FCUTABLE = [
```

Comment:
Is there any document to explain this? Please also consider to explain the column and raw. Thanks!

Reply:
I agree that a comment preceding the list would be helpful.

Also I gently suggest making it a tuple of tuples (parenthesis instead of square braces) to make both the list and the items in the list immutable.

Reply:
This will be most likely replaced by some array of objects, or Numpy named array. The whole notion of presenting indices as extra values is bad. This is currently not used, just taken from FA. Fix shall be provided in https://jira.lsstcorp.org/browse/DM-32672

Reply:
I just noticed the values FCUTABLE_ZPOSITION = 3, etc. I assume those are indices into the arrays of values, and will be replaced. But if you end up retaining them, I strongly suggest an enum.

A class with named attributes for the different collections of values would be a nice way to handle the values. That could be a SimpleNamespace (which is basically just a dict with .foo instead of ["foo"] access) or a dataclass (same result, different way of defining it) would save some boilerplate.